templates/partners/find-a-partner.html

```diff
@@ -0,0 +1,496 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom">
```

Comment:
I don't think you need the `is-deep` class, you're removing the padding it adds with the utility classes.